server/repository/odp/deleteDraft.ts

```diff
@@ -0,0 +1,17 @@
+import { getOdpNationalClasses, wipeNationalClassIssues } from '@server/repository/odpClass/odpClassRepository'
+import { OdpService } from '@server/service'
+import { getAndCheckOdpCountryId } from './getAndCheckOdpCountryId'
+
+export const deleteDraft = async (client: any, odpId: any, user: any) => {
+  const countryIso = await getAndCheckOdpCountryId(client, odpId, user)
+  const actualRes = await client.query('SELECT actual_id FROM odp WHERE id = $1', [odpId])
+  const actualId = actualRes.rows[0].actual_id
+  if (actualId) {
+    await client.query('UPDATE odp SET draft_id = null WHERE id = $1', [odpId])
+    const odpVersionId = await OdpService.getOdpVersionId(client, odpId)
+    const odpClasses = await getOdpNationalClasses(client, odpVersionId)
+    return wipeNationalClassIssues(client, odpId, countryIso, odpClasses)
+  }
+
+  return OdpService.deleteOdp(client, odpId, user)
+}
```

Comment:
repository cannot use service.

plus these operations should be executed in transactions created in Service.
should we talk about this?

Reply:
just don't use service in repository for now.
will make a new task for transactions